src/effects/native/parametriceqeffect.h

```diff
@@ -19,7 +19,7 @@
 // The ParamtricEQEffect models the mid bands from a SSL Black EQ (242)
 // with a gentle parameter range, as requested here:
 // https://www.mixxx.org/forums/viewtopic.php?f=3&t=9239&p=33312&hilit=SSL#p33312
-// The main use case is to tweak the room or recording sound, which is hard to acheve
+// The main use case is to tweak the room or recording sound, which is hard to achieve
 // with the sharp and wide curves of the mixing EQs.
```

Comment:
This information should not be buried in a source code comment. The user-facing description should make it clear what the effect is for.